Charades_Game_In_JavaScript/Charades Game In JavaScript final blog.md

```diff
@@ -0,0 +1,350 @@
+
+You can play around with the design of the button in the corresponding CSS file "styles.css." Now let's focus on adding the effect when the button is clicked.
+
+**in HTML:**
+First, the tag `<button>` can have an **"onclick"** event, which calls the function `start()`:
+
+![](https://i.imgur.com/aFkiBAS.png)
+
+To make the button work, we need to write the function in JavaScript.
+
+**in JavaScript:**
+We want to go to the next page of the game when the start button is clicked.
+
+Our `function start() {}` would only have one line to jump to the next HTML page using `window.location`.
+
+![](https://i.imgur.com/eV0yayP.png)
+
+
+**in CSS:**
+Here is an example design of the start button, but you can customize yours by playing around with the features.
+
+![](https://i.imgur.com/hpqyiQl.png)
+
+
+
+### Tip: Linking Two HTML Files
+As we jump from the starting page to the next page, we switch into another HTML file. In order to make this happen, we need to add another line of code in our HTML file.
+
+Use the tag `<a href="{{YOUR_NEXT_HTML_NAME}}"></a>` under the `<head>` tag in your starting page's HTML. Now check out the page — the start button should be working!
+
+![](https://i.imgur.com/HSesBXq.png)
+
+
+
+### Ending Page
+
+In the [live demo](https://cotton-broadleaf-pufferfish.glitch.me/), we see that the ending page is just a page with one single line of words to show a finishing message.
+
+![](https://i.imgur.com/bBJIeDc.jpg)
+
+To do this, we can simply write in the HTML file.
+A simple header shows the message, but we want it to be wrapped by a section so the text is centered.
+
+```
+<body>
+    <section>
+        <h1>Times up!</h1>
+    </section>
+</body>
+```
+
+Since this is a very simple page, we don't need a script file to add functions to it. Instead of using a separate CSS file to style the contents, we can simply add `style="..."` in each tag to customize the style.
+
+
+
+## Choosing the Game Rules
+
+![](https://i.imgur.com/4yewZS5.png)
+
+The start button redirects us to the second page where we can choose the version of the game we want to play. One is called "Limiting Time," where you get 30 seconds per card for a set number of words, and the other is "How Many Words," where you get 5 minutes total and you skip through cards as fast as you want to.
+
+***Want to play the game right now? Skip this page! Jump directly to the "Creating Animated Word Cards" section about your preferred version!***
+
+**in HTML:**
+On this page, we want to have two buttons which redirect to two different HTML pages. The buttons are created in the same way as the start button on the starting page. We add a `<p></p>` to create the text box under each button.
+
+Create two `<div>` with class names "v1" and "v2" for the two buttons under the `<section>` first. These containers will allow us to set the positions of the buttons and the text box in CSS later.
```

Comment:
create two '<div>' tags? Please look through where you cite the tags and names and makes sure you say that they are tags, etc.